temba/flows/views.py

```diff
@@ -1076,13 +1076,7 @@ def get_gear_links(self):
                 )
 
             # Weni Addition
-            links.append(
-                dict(
-                    title=_("Copy UUID"),
-                    style="button-secondary",
-                    copyUuid=_("Copy UUID")
-                )
-            )
+            links.append(dict(title=_("Copy UUID"), style="button-secondary", copyUuid=_("Copy UUID")))
```

Comment:
why this is shown on this PR? it wasn't `merged` before?

Reply:
I just ran code_check.py